colossalai/shardformer/__init__.py

```diff
@@ -1 +1,5 @@
+from .modeling.chatglm2_6b.configuration_chatglm import ChatGLMConfig
```

Comment:
Remove chatglm related imports may resolve the error during doc building

Reply:
removed